src/ppx/application.cpp

```diff
@@ -1114,16 +1090,9 @@ void Application::UpdateStandardSettings()
 {
     mSettings.headless = mStandardOpts.pHeadless->GetValue();
 
-    // If command line argument provided width and height
-    auto       resolution        = mStandardOpts.pResolution->GetValue();
-    const bool hasResolutionFlag = (resolution.first > 0 && resolution.second > 0);
-    if (hasResolutionFlag) {
-        mSettings.window.width  = resolution.first;
-        mSettings.window.height = resolution.second;
-    }
-
 #if defined(PPX_BUILD_XR)
-    resolution = mStandardOpts.pXrUiResolution->GetValue();
+    auto resolution = mStandardOpts.pResolution->GetValue();
```

Comment:
why are we setting it like this if we are going to overwrite it in the next line? Not sure if I am missing something

Reply:
Just a mistake when I refactorized this. Removing the first line. Thanks for catching!